plugin/adsk/scripts/mayaUsd_preferenceTab.mel

```diff
@@ -392,10 +408,22 @@ global proc mayaUsd_PrefHoldState(string $mode)
 	};
 	prefsHoldOptionVars($prefOptionVars, $mode); 
 
-	if ($mode == "remove") {
-		// Finalize preference changes into MayaUsd.
+    if ($mode == "save") {
+        // Preferences dialog was opened, make sure we don't have this transient optionVar.
+        optionVar -remove mayaUsd_PrefUntexturedModeChanged;
+    }
+
+	if ($mode == "restore") {
```

Comment:
Indentation seems off here..

Reply:
Sorry it is a mix of tabs/spaces. I'll fix it.